variables.tf

```diff
@@ -191,3 +191,9 @@ variable "hieradata_repo_user" {
 variable "control_vshn_net_token" {
   type = string
 }
+
+variable "team" {
+  type        = string
+  description = "Team to assign the load balancers to in Icinga. All lower case."
+  default     = ""
```

Comment:
I think we should create a follow-up to make the `team` variable mandatory after all existing setups have been updated to make use of it.